src/presentation/rest/UsersController.ts

```diff
@@ -78,6 +87,38 @@ export class UsersController extends Controller {
 
         return;
     }
+
+    @Get('activate/{generatedUUID}')
+    @SuccessResponse('200', 'User Activated')
+    @Response('404', 'Link is not valid or expired')
+    public async activateUser(@Path() generatedUUID: string): Promise<void> {
+        await this.usersService.activateUser(generatedUUID);
+        this.setStatus(200);
+    }
+
+    @Post('{userId}/sendactivationlink')
+    @SuccessResponse('200', 'Sended')
+    public async sendActivationLink(@Path() userId: number, @Request() request: ExRequest): Promise<void> {
+        try {
+            const ACTIVATION_PATH = request.get('host') + '/api/users/activate/';
+
+            const createdUser = await this.usersService.get(userId);
+
+            const personalUUID = await this.usersService.createPersonalActivationUUID(userId);
+
+            const message = await this.emailService.createActivationMessage(ACTIVATION_PATH + personalUUID);
+
+            await this.emailService.sendEmail(createdUser.mail, message);
+
+            this.setStatus(200);
+            return;
+        } catch (error) {
+            throw error;
+        }
```

Comment:
Możesz to wrzucić do metody w pliku `UsersService` i w kontrolerze tylko wywołać taką metodę